app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -188,34 +188,34 @@ def self.host_inv_to_hashes(inv, ems_inv, storage_uids, cluster_uids)
           end
 
           new_result = {
-            :type             => %w(esx esxi).include?(product_name.to_s.downcase) ? "ManageIQ::Providers::Vmware::InfraManager::HostEsx" : "ManageIQ::Providers::Vmware::InfraManager::Host",
-            :ems_ref          => mor,
-            :ems_ref_obj      => mor,
-            :name             => hostname,
-            :hostname         => hostname,
-            :ipaddress        => ipaddress,
-            :uid_ems          => uid_ems,
-            :vmm_vendor       => vendor,
-            :vmm_version      => product["version"],
-            :vmm_product      => product_name,
-            :vmm_buildnumber  => product["build"],
-            :connection_state => connection_state,
-            :power_state      => connection_state != "connected" ? "off" : (maintenance_mode.to_s.downcase == "true" ? "maintenance" : "on"),
-            :admin_disabled   => config["adminDisabled"].to_s.downcase == "true",
-            :asset_tag        => asset_tag,
-            :service_tag      => service_tag,
-            :failover         => failover,
-            :hyperthreading   => config.fetch_path("hyperThread", "active").to_s.downcase == "true",
-
-            :ems_cluster      => cluster_uids_by_host[mor],
-            :operating_system => host_inv_to_os_hash(host_inv, hostname),
-            :system_services  => host_inv_to_system_service_hashes(host_inv),
-
-            :hardware         => hardware,
-            :switches         => switches,
-            :storages         => storages,
-
-            :child_uids       => rp_uids,
+            :type               => %w(esx esxi).include?(product_name.to_s.downcase) ? "ManageIQ::Providers::Vmware::InfraManager::HostEsx" : "ManageIQ::Providers::Vmware::InfraManager::Host",
+            :ems_ref            => mor,
+            :ems_ref_obj        => mor,
+            :name               => hostname,
+            :hostname           => hostname,
+            :ipaddress          => ipaddress,
+            :uid_ems            => uid_ems,
+            :vmm_vendor_display => vendor,
```

Comment:
I feel like the parsers should still set the column directly, so this should still set `vmm_vendor`